app/models/article.rb

```diff
@@ -309,6 +309,7 @@ def remove_algolia_index
   def touch_by_reaction
     async_score_calc
     index!
+    index_to_elasticsearch
```

Comment:
Just curious: what does `index!` do versus `index_to_elasticsearch`? Just grepped for `index!` and couldn't find it in the codebase so wanted to know more about the magic that's happening here :)

Reply:
`index!` is for Algolia and given to us from the Algolia gem we use. It will be removed once we are switched to Elasticsearch. For now, we have to keep both in sync.